src/components/common/ListMapSelector.tsx

```diff
@@ -28,11 +28,14 @@ export type ListMapSelectorProps = {
 };
 
 export default function ListMapSelector({ view, onView }: ListMapSelectorProps): JSX.Element {
+  const [initialView] = useState<View>(view);
```

Comment:
The `view` prop feels a bit overloaded here. It's responsible for determining the initial view (whatever it was first set to) and the current view. Would it be cleaner to have a separate `defaultView` prop?